src/ks_rank_sensitivity.jl

```diff
@@ -0,0 +1,131 @@
+raw"""
+
+    KSRank(; n_dummy_parameters::Int = 50, acceptance_threshold::Union{Function, Real} = mean)
+
+- `n_dummy_parameters`: Number of dummy parameters to add to the model, used for sensitivity hypothesis testing and to check the amount of samples. Defaults to 50.
+- `acceptance_threshold`: Threshold or function to compute the threshold for defining the acceptance distribution of the sensitivity outputs. The function must be of the signature f(Y) and return a real number. 
+   Defaults to the mean of the sensitivity values.  
+
+## Method Details
+
+The KSRank method is a monte-carlo based technique for performing nonparametric global sensitivity analysis. The method is based on the Kolmogorov-Smirnov (KS) test, which is a nonparametric test of the equality of continuous, 
+one-dimensional probability distributions that can be used to compare a sample with a reference probability distribution. The method is used to compare the acceptance and rejection distributions of the sensitivity outputs, which are
+calculated by comparing the sensitivity outputs with a threshold value. The sensitivity for each parameter is then given by
+
+```math
+S_{i} = \sup_{j} |F_i - (1 - F_i)|
```

Comment:
What's `j` here? and the equations looks off should one of it be F_j instead of F_i?

Reply:
Good catch. `j` is the sample in the distribution. Indeed, it should even be $F_{i,j}$ each parameter has its own cumulative distribution. This is updated in the lates commit.